client/post-editor/editor-title/style.scss

```diff
@@ -1,10 +1,6 @@
 .editor-title {
```

Comment:
Is there anything we could simplify markup-wise here? Some of these nested elements were needed for correctly rendering the fade effect, which we don't need anymore.